src/view.coffee

```diff
@@ -1642,12 +1646,46 @@ define ['droplet-helper', 'droplet-draw', 'droplet-model'], (helper, draw, model
 
         # Blocks have a shape including a lego nubby "tab", and so
         # they need to be at least wide enough for tabWidth+tabOffset.
+
+        @extraWidth = 0
+        if 'add-button' in @model.classes
+          @extraWidth += @view.opts.buttonWidth + @view.opts.buttonPadding
+
+        if 'subtract-button' in @model.classes
+          @extraWidth += @view.opts.buttonWidth + @view.opts.buttonPadding
+
         for size, i in @minDimensions
           size.width = Math.max size.width,
               @view.opts.tabWidth + @view.opts.tabOffset
+          size.width += @extraWidth
```

Comment:
For a multiline block, you're adding @extrawidth to every line.  The +/- button should be on the last line, so should just apply there.